### PR TITLE
fixmes

### PR DESCRIPTION
**Pull request by Kreyren**

### CODE REVIEW

**Comment by Kreyren** [comment]
@Uniminin now go and fix things

File: make.conf [hunk 1]
```diff
@@ -2,16 +2,48 @@
 # built this stage.
 # Please consult /usr/share/portage/config/make.conf.example for a more
 # detailed example.
+# FIXME: Add -march flag based on https://wiki.gentoo.org/wiki/Safe_CFLAGS
 COMMON_FLAGS="-O2 -pipe"
```
Comment (Kreyren) [inline make.conf]: ping

File: make.conf [hunk 2]
```diff
@@ -2,16 +2,48 @@
...
 CFLAGS="${COMMON_FLAGS}"
 CXXFLAGS="${COMMON_FLAGS}"
 FCFLAGS="${COMMON_FLAGS}"
 FFLAGS="${COMMON_FLAGS}"
+CPU_FLAGS_X86=
```
Comment (Kreyren) [inline make.conf]: ping

File: make.conf [hunk 3]
```diff
@@ -2,16 +2,48 @@
...
+CHOST=
```
Comment (Kreyren) [inline make.conf]: ping

File: make.conf (hunk 1 above)
Comment (Uniminin) [reply make.conf]: ok

File: make.conf [hunk 4]
```diff
@@ -2,16 +2,48 @@
...
+CBUILD=
```
Comment (Kreyren) [inline make.conf]: ping

File: make.conf [hunk 5]
```diff
@@ -2,16 +2,48 @@
...
+VIDEO_CARDS=
```
Comment (Kreyren) [inline make.conf]: ping

**Pull request merged**